docs/src/components/DocLayout/index.tsx

```diff
@@ -52,11 +53,15 @@ interface Props {
     url: string;
   }>;
   custom?: boolean;
+  noTopBar?: boolean;
+  hasStorybook?: boolean;
+  storybookLink?: false | string;
```

Comment:
set it to a boolean and when it's pass down to this component add `!!storybookLink or Boolean(storybookLink)`

Reply:
Which type are you referring to? `storybookLink`? I think it's better to have it like this so we can never pass `true` to the prop

Reply:
but storybookLInk can be only a string or null, no? hasStorybook is supposed to be a boolean. In schema it's type string, so if you will pas false there in yaml file it should throw error

Reply:
Oh, yes, it should be string. It's a leftover from previous implementation. Fixed